### PR TITLE
📝 Add Authoring commits Git section

### PR DESCRIPTION
**Pull request by smolijar**

### CODE REVIEW

File: git/guides/authoring-commits.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# 👨‍💻 Authoring commits
+
+## User identity
+
+1. Always user your full `name`
```
Comment (davidbilik) [inline git/guides/authoring-commits.md:5]: typo :(

File: git/guides/authoring-commits.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+   Features are not atomic and abiding this rule might be difficult. Try to divide complex features into incremental changes, prefer dangling unused code (to be integrated later) to compilation error or app start failure.
+   Following this rule pays off when doing version rollback or using `git bisect` to detect code breaking changes.
+
+2. **Follow the single responsibility principle.**
+   Assume each revision you make to have the possibility to introduce a bug. Wen you and your team identify the commit, following the SRP helps you in various ways. You (A) easily identify the cause of the incorrect behavior, since your commit does nothing else than the changes related to that behavior and (B) if you find out it was not such a great idea, you can `revert` the commit and not have to worry about "the good parts" of the commit you would like to keep and (C) you are more likely to avoid collisions with other collaborators, who might have had the same idea to apply unrelated changes.
+
+## Feature branch history
+
+Try to make history in your feature branch a self-explanatory changelog. It will me more pleasant to review.
```
Comment (davidbilik) [inline git/guides/authoring-commits.md:28]: typo :(

File: git/guides/authoring-commits.md (hunk 1 above)
Comment (smolijar) [reply git/guides/authoring-commits.md:5]: Thanks, fixed in #3

File: git/guides/authoring-commits.md (hunk 2 above)
Comment (smolijar) [reply git/guides/authoring-commits.md:28]: Thanks, fixed in #3

**Pull request merged**